src/libraries/System.Linq/tests/System.Linq.Tests.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+    <DefineConstants Condition="'$(TargetOS)' == 'Browser'">$(DefineConstants);OUTERLOOP</DefineConstants>
```

Comment:
This would need Browser in the list of the TargetFrameworks above to work well.

Reply:
I suggest to unilaterally outer loop instead